x/epochs/types/genesis.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+// DefaultIndex is the default capability global index
+const DefaultIndex uint64 = 1
+
+// DefaultGenesis returns the default Capability genesis state
+func DefaultGenesis() *GenesisState {
+	return &GenesisState{}
```

Comment:
I think, it's required to add at least an epochInfo to this

Reply:
It's added